registers/dump/main.go

```diff
@@ -29,8 +76,20 @@ func main() {
 	}
 	log.Printf("%#v", r)
 
-	err = r.GetEntries(&dumper{})
+	g, err := grpc.Dial(*trillianLog, grpc.WithInsecure())
```

Comment:
At first glance, I got confused why the connection was created after the request had already been made! Then I expanded the diff and realized this was a different function and we're back in `main` now.

Perhaps if "Step 1" had laid things out so that the parts filled here were written there as stubbed out methods, then the diff would've been easier to understand/follow.

wdyt?